invokeai/app/api/routers/images.py

```diff
@@ -84,17 +99,23 @@ async def upload_image(
 
     filename = f"{uuid.uuid4()}_{str(int(datetime.now(timezone.utc).timestamp()))}.png"
 
-    (image_path, thumbnail_path, ctime) = ApiDependencies.invoker.services.images.save(
+    (image_name, thumbnail_name, ctime) = ApiDependencies.invoker.services.images.save(
         ImageType.UPLOAD, filename, img
     )
 
     invokeai_metadata = ApiDependencies.invoker.services.metadata.get_metadata(img)
 
     res = ImageResponse(
         image_type=ImageType.UPLOAD,
-        image_name=filename,
-        image_url=f"api/v1/images/{ImageType.UPLOAD.value}/{filename}",
-        thumbnail_url=f"api/v1/images/{ImageType.UPLOAD.value}/thumbnails/{os.path.splitext(filename)[0]}.webp",
+        image_name=image_name,
+        image_url=request.url_for(
```

Comment:
Is there a reason to use `request.url_for` instead of a method on the image storage service? That would make it more flexible for other services that may need to utilize different URLs to access the image

Reply:
No good reason. That makes sense. I think I've implemented this in a reasonable way in the last commit to this feature branch. @Kyle0654